tsconfig.json

```diff
@@ -1,26 +1,31 @@
 {
-  "compilerOptions": {
-    "target": "es6",
-    "module": "commonjs",
-    "sourceMap": true,
-    "moduleResolution": "node",
-    "allowSyntheticDefaultImports": true,
-    "experimentalDecorators": true,
-    "esModuleInterop": true,
-    "emitDecoratorMetadata": true,
-    "strict": true,
-    "skipLibCheck": true,
-    "typeRoots": [
-      "node_modules/@types",
-      "types"
+    "compilerOptions": {
+        "target": "es6",
+        "module": "commonjs",
+        "sourceMap": true,
+        "moduleResolution": "node",
+        "allowSyntheticDefaultImports": true,
+        "experimentalDecorators": true,
+        "esModuleInterop": true,
+        "emitDecoratorMetadata": true,
+        "strict": true,
+        "skipLibCheck": true,
+        "typeRoots": [
+            "node_modules/@types",
+            "types"
+        ],
+        "strictPropertyInitialization": false
+    },
+    "include": [
+        "src/**/*.ts",
+        "test/**/*.ts",
+        "src/Migrations/20191224091606-create-node.js",
+        "src/Migrations/20180827100753-create-example.js",
+        "src/Migrations/20191224091606-create-node.js",
```

Comment:
Migration file is missing, not commited?

Reply:
They are excluded by default, should I remove the "src/Migrations" exclusion?

Reply:
Hmm, I think because of the latest changes you can move `src/Migrations` to include. Might need to set flag `allowJs` to true.